build.gradle

```diff
@@ -30,7 +30,7 @@ buildscript {
         gradleVersion = '3.2.0-rc03'
         gsonVersion = '2.8.2'
         junitVersion = '4.12'
-        kotlinVersion = '1.3-M2'
+        kotlinVersion = '1.3.0-rc-57'
```

Comment:
It works in "eap" repo.

However, it is really strang that this version is in "eap" not in "dev". 
Take a look at [here, 1.3.0-rc-46](https://mvnrepository.com/artifact/org.jetbrains.kotlin/kotlin-gradle-plugin/1.3.0-rc-46)